src/pages/forms/add-admin-form.html

```diff
@@ -0,0 +1,584 @@
+              <i class="menu-arrow"></i>
+            </a>
+          </li>
+          <!------------------------------------------------------------------------------------------->
+          <li class="nav-item nav-category">help</li>
+          <li class="nav-item">
+            <a class="nav-link" href="http://bootstrapdash.com/demo/star-admin2-free/docs/documentation.html">
+              <i class="menu-icon mdi mdi-file-document"></i>
+              <span class="menu-title">Documentation</span>
+            </a>
+          </li>
+        </ul>
+      </nav>
+<!--------------------------------------------------------End of SideBar------------------------------------------------->
+<!-------------THe Form----------->
+        <div class="main-panel">
+          <div class="content-wrapper">
+            <div class="row">
+              <!-- Changed the class from "col-md-6" to "col-12" -->
+              <!-- "col-md-6" means the column takes up half the width on medium devices and larger -->
+              <!-- "col-12" means the column takes up the full width on devices of all sizes -->
+              <div class="col-12 grid-margin stretch-card"> <!-- This line was modified -->
+                <div class="card">
+                  <div class="card-body">
+                    <h4 class="card-title">Add Admin form</h4>
+                    <p class="card-description">Add a new admin user to the dashboard</p>
+                    <form class="forms-sample">
+                      <div class="form-group">
+                        <label for="exampleInputUsername1">Username</label>
+                        <input type="text" class="form-control" id="exampleInputUsername1" placeholder="Username">
+                      </div>
+                      <div class="form-group">
+                        <label for="exampleInputEmail1">Email address</label>
+                        <input type="email" class="form-control" id="exampleInputEmail1" placeholder="Email">
+                      </div>
+                      <!-- Added mobile number input ---- Type "number" to ensure numerical input -->         
+                      <div class="form-group">
+                         <label for="exampleInputMobile1">Mobile</label>
+                         <input type="number" class="form-control" id="exampleInputMobile1" placeholder="Mobile Number">
+                      </div>
+                      <div class="form-group">
+                        <label for="exampleInputPassword1">Password</label>
+                        <input type="password" class="form-control" id="exampleInputPassword1" placeholder="Password">
+                      </div>
+                      <div class="form-group">
+                        <label for="exampleInputConfirmPassword1">Confirm Password</label>
+                        <input type="password" class="form-control" id="exampleInputConfirmPassword1"
+                          placeholder="Password">
```

Comment:
Actually good job for this, even though i didn't mention it.
Thanks for your details